### ISSUE TITLE
Documentation of risk metric score calculations

### ISSUE THREAD

**Issue by LDSamson** [pharmaR/riskmetric#327]
The description of how individual metrics are calculated is hard to find. I had to dive deep into the source code of the package to, for example, find how the score for 'package metric dependencies' is calculated. Is there any documentation that I did not find yet about this topic?

Maybe the standard scores can be included in a vignette, including the calculation method and rationale? That would be really helpful.

**Comment by dgkf** [comment]
Yeah, this sounds like we could definitely improve the discoverability here.

We could add some dynamically generated roxygen docs to `?metric_score` like we do for [`?pkg_assess`](https://github.com/pharmaR/riskmetric/blob/master/R/pkg_assess.R#L147C1-L147C41). We already have the description attributes to support `{riskassessment}`, so we can probably do something similar here.

Of course, if you think the docs themselves for the specific methods could be improved, then that's always a place where we welcome contributions!
